sentry-android/src/main/java/io/sentry/android/AndroidSerializer.java

```diff
@@ -18,6 +19,11 @@
           .registerTypeAdapter(SentryId.class, new SentryIdDeserializerAdapter())
           .registerTypeAdapter(Date.class, new DateSerializerAdapter())
           .registerTypeAdapter(Date.class, new DateDeserializerAdapter())
+          .registerTypeAdapter(
```

Comment:
You should evaluate usage of `TypeAdapterFactory` here instead of manually adding type adapters for each class that must support unknown properties.